File: Domain/DTO/History.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Domain.DTO
{
    public class History
    {
        public int Id { get; set; }
        public int EmployeeId { get; set; }
        public EmployeeDTO Employee { get; set; }

        public int PositionId { get; set; }
        public PositionDTO Position { get; set; }
        public double Salary { get; set; }

        public DateTime Hired { get; set; } 
        public DateTime ? Fired { get; set; }


    }
}

File: PL/ClientApp/src/app/shared/models/history.ts
import {Employee} from "./employee"
import {Position} from "./position"
import { from } from "rxjs";
export class History {
    Id :number;
    EmployeeId :number;
    Employee :Employee;
    PositionId : number;
    Position :Position;

    Hired :Date;
    Fired :Date;
    Salary :number;




}